packages/docs-app/src/examples/select-examples/multiSelectExample.tsx

```diff
@@ -124,7 +124,7 @@ export class MultiSelectExample extends React.PureComponent<IExampleProps, IMult
                     // we may customize the default filmSelectProps.items by
                     // adding newly created items to the list, so pass our own
                     items={this.state.items}
-                    noResults={<MenuItem disabled={true} text="No results." />}
+                    noResults={<MenuItem disabled={true} text="No results." roleConfig="listoption" />}
```

Comment:
this seems to be common enough that we should add a note to the `noResults` prop documentation so that users remember to set `roleKind`

Reply:
good idea, added